test/main.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
add the license to all files wherever required.
you can use the cortx code license, and check what needs to change.
in case of any questions regarding the license talk with @shaileshsvc or @ajaysrivas

Reply:
Added license.